testing/testcafe/tests/dataGrid/grouping/grouping.ts

```diff
@@ -1,9 +1,12 @@
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+/* eslint-disable spellcheck/spell-checker */
```

Comment:
Looks like we'll write it very often, let's add `axe` word to spell-checker dictionary in eslintrc.json, and move it to `eslint-config-devextreme` in future